lib/tutorials/serving-files.md

```diff
@@ -1,18 +1,35 @@
 ## Serving static files
 
-Inevitably while building any web application, the need arises to serve a simple file from disk. hapi provides a pair of built-in handlers, as well as a reply method, to make doing so very simple.
+_This tutorial is compatible with hapi v9.x.x._
+
+Inevitably while building any web application, the need arises to serve a simple file from disk. There's a hapi plugin called [inert](https://github.com/hapijs/inert) that adds this functionality to hapi through the use of additional handlers.
+
+First you need to install and add inert as a dependency to your project:
+
+`npm install --save inert`
 
 ## reply.file()
 
 First, to use the reply method:
 
 ```javascript
-server.route({
-    method: 'GET',
-    path: '/picture.jpg',
-    handler: function (request, reply) {
-        reply.file('/path/to/picture.jpg');
+server.register(Inert, function (err) {
```

Comment:
I think you should mention what `Inert` refers to, or just use `require('inert')` in `server.register()`.